### DISCUSSION TITLE
Trouble flashing the SD card th the JetBot file

### DISCUSSION THREAD

**Discussion by Hmk5** [NVIDIA-AI-IOT/jetbot#435] [Q&A]
I am having trouble flashing the SD card th the JetBot file. I have tried all the files on [https://jetbot.org/master/software_setup/sd_card.html](url) using my Jetson nano 2GB dev kit and non seem to work.  Ove version gave me a black screen when trying to boot, the other gave me a long repetitive message "no acknowledge from address 0x3c", one would bring up the Nvidia logo but then shut down. 

I have bought a new SD card and also couldn't flash the jetbot file but luckily i have mangesd to flash the "jetson-nano-2gb-jp451-sd-card-image" regular file from the Nvidia site and boot properly. 

Any suggestions? How can use the Jetbot environment in this case? 

I have used a sandisk ultra 64GB A1 SD card to begin with and then the sandisk 64GB A2 SD card. 

Thanks for the help

**Comment by santaimpersonator** [comment]
For the 2GB board, with Jetpack 4.5 you'll need to boot the board with the plain image before using the prebuilt image (as mentioned in the call out at the bottom of Step 1).
